scripts/drtprod

```diff
@@ -27,7 +27,8 @@ fi
 
 case $1 in
   "start")
-    if [[ "$*" != *"--restart"* ]]; then
+    cluster="$2"
+    if [[ $cluster != "drt-large" && "$*" != *"--restart"* ]]; then
```

Comment:
This is to avoid the command to have `--restart` and `--restart=false` both in case of drt-large

Reply:
Done